WooCommerce/Classes/POS/Models/PointOfSaleOrderState.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+enum PointOfSaleOrderState: Equatable {
+    case idle
+    case syncing
+    case loaded(PointOfSaleOrderTotals)
```

Comment:
Note this change; we include the order totals here rather than the view getting them from the order separately.